src/durabletask-preview/azext_durabletask_preview/azext_metadata.json

```diff
@@ -0,0 +1,5 @@
+{
+    "azext.isPreview": true,
+    "name": "durabletask-preview",
```

Comment:
Is this `name` auto-generated by codegen tool? usually we don't need it in this metadata file.